app/assets/src/components/ui/controls/dropdowns/MultipleDropdown.jsx

```diff
@@ -36,50 +40,57 @@ class MultipleDropdown extends React.Component {
     let checkedOptions = this.props.value || this.state.value;
     for (let option of this.props.options) {
       ret.push(
-        <BaseDropdown.Item
+        <BareDropdown.Item
           key={option.value}
           data-value={option.value}
-          onClick={this.optionClicked.bind(this)}
+          onClick={this.handleItemClicked}
         >
-          <span className="text">
-            <input
-              type="checkbox"
-              checked={checkedOptions.indexOf(option.value) > -1}
-              readOnly
-            />
-            {option.text}
-          </span>
-        </BaseDropdown.Item>
+          <Checkbox
```

Comment:
Can we create a common component for checkbox items?
Could avoid the duplicate below in multiple and nested dropdowns...